package.json

```
@@ -165,7 +172,8 @@
   "workspaces": {
     "packages": [
       ".",
-      "wrappers/*"
+      "wrappers/*",
+      "examples/*"
```

I'm not sure if I'm getting the right intention about the project, so correct me if I'm wrong, but probably the entire `examples` directory should be considered a separate project, not all the subdirectories inside it.

```suggestion
      "examples"
```